src/main/kotlin/baseball/domain/Game.kt

```diff
@@ -0,0 +1,12 @@
+package baseball.domain
+
+class Game(private val numbersGenerator: NumbersGenerator = RandomNumbersGenerator()) {
```

Comment:
이걸 의도하신 게 맞겠죠? 아니면 default 값을 지정해두신 건가요? 코틀린 어렵군뇨
```suggestion
class Game(private val numbersGenerator: NumbersGenerator) {
```

Reply:
디폴트값이요 헷